### PR TITLE
feat(AHWR-233): add you can claim for multiple species page, behind feature flag

### PR DESCRIPTION
**Pull request by bendef-24**

### CODE REVIEW

File: app/routes/endemics/you-can-claim-multiple.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+        session.setFarmerApplyData(
+          request,
+          agreeSameSpecies, // NOTE AHWR-427 switch to agreeMultipleSpecies, once MI report supports it
+          status.storedAnswer
+        )
+
+        if (status.value === agreeStatusValue) {
+          return h.redirect(nextPage)
+        } else {
```
Comment (RobCatton) [inline app/routes/endemics/you-can-claim-multiple.js:60]: Minor thing, but I dont think we need the else here. Could help with readability if we removed it and added a line to separate when `status.value === agreeStatusValue `and when it doesnt.

Comment (bendef-24) [reply app/routes/endemics/you-can-claim-multiple.js:60]: Agreed, will change shortly

File: app/routes/endemics/you-can-claim-multiple.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+module.exports = [
+  {
+    method: 'GET',
+    path: pageUrl,
+    options: {
+      handler: async (request, h) => {
+        const organisation = session.getFarmerApplyData(request, organisationKey)
+        return h.view(endemicsYouCanClaimMultiple, {
+          backLink,
+          agreementStatus: agreementStatuses,
```
Comment (mattyod) [inline app/routes/endemics/you-can-claim-multiple.js:39]:
```suggestion
          agreementStatus,
```

File: app/routes/endemics/you-can-claim-multiple.js (hunk 1 above)
Comment (mattyod) [reply app/routes/endemics/you-can-claim-multiple.js:60]: yep no need for an else if you are making an early return.

File: app/views/endemics/you-can-claim-multiple.njk [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+      <p class="govuk-body">You can do a maximum of 3 reviews and 3 follow-ups per species. You must do all your reviews and follow-ups by 19 June 2027.</p>
+      <p class="govuk-body">If you did a review in the old annual health and welfare service, it will not count towards your limit.</p>
+      <p class="govuk-body">You need to make a separate claim for each species.</p>
+      <br>
+
+      <form method="POST" autocomplete="off" novalidate="novalidate" id ="submitYouCanClaimMultipleForm">
+        <input type="hidden" name="crumb" value="{{crumb}}"/>
+        <div class="govuk-button-group">
+          {{ govukButton({
+          attributes: { "aria-label": agreementStatus[0].text + " - " + pageHeadingText },
```
Comment (mattyod) [inline app/views/endemics/you-can-claim-multiple.njk]: Do we need aria labels when the button has text already? In this case it's going to read out the same text as the button has along with the entire page heading for each button.

File: app/routes/endemics/you-can-claim-multiple.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+const pageUrl = `${urlPrefix}/${endemicsYouCanClaimMultiple}`
+const backLink = `${urlPrefix}/${endemicsCheckDetails}`
+const nextPage = `${urlPrefix}/${endemicsNumbers}`
+
+const agreeStatusValue = 'agree'
+const agreementStatuses = [
+  {
+    value: agreeStatusValue,
+    text: 'I agree',
+    storedAnswer: 'yes'
```
Comment (mattyod) [inline app/routes/endemics/you-can-claim-multiple.js:21]: Not sure I understand why we use a different value for the form components from what we store. Would it work if we just set the values for the two buttons to be 'yes' and 'no'?

File: app/views/endemics/you-can-claim-multiple.njk (hunk 3 above)
Comment (bendef-24) [reply app/views/endemics/you-can-claim-multiple.njk]: I did by example on this, copied from Reviews page, will have a think can get back to you

File: app/routes/endemics/you-can-claim-multiple.js (hunk 4 above)
Comment (bendef-24) [reply app/routes/endemics/you-can-claim-multiple.js:21]: Agreed, was on my list but forgot to action, will get back to you but can't see why they would need to be different

Comment (bendef-24) [reply app/routes/endemics/you-can-claim-multiple.js]: Updated code to use button value instead of storedAnswer

**Review by RobCatton — APPROVED** [review]

**Pull request merged**